### PR TITLE
CLI: Fix error on dir or file that name is a number

### PR DESCRIPTION
**Pull request by fisker**
Prettier can't work if a dir/file named `1` passed to CLI

- [x] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory)
- [x] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/pr-XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

**Review by evilebottnawi — APPROVED** [review]

File: src/cli/util.js [hunk 1]
```diff
@@ -1010,7 +1010,7 @@ function updateContextArgv(context, plugins, pluginSearchDirs) {
   const argv = minimist(context.args, minimistOptions);
 
   context.argv = argv;
-  context.filePatterns = argv._;
+  context.filePatterns = argv._.map((file) => String(file));
```
Comment (sosukesuzuki) [inline src/cli/util.js:1013]: Is this the same behavior?
```suggestion
  context.filePatterns = argv._.map(String);
```

Comment (fisker) [reply src/cli/util.js:1013]: Yes, but normally, I follow [unicorn/no-fn-reference-in-iterato](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/no-fn-reference-in-iterator.md) rule.

Comment (sosukesuzuki) [reply src/cli/util.js:1013]: Is the rule enable on this project? I don't have strong opinion for this coding style, but I think the rules should be enabled for uniformity.

Comment (fisker) [reply src/cli/util.js:1013]: I can do this in another PR.

**Review by sosukesuzuki — APPROVED** [review]

**Pull request merged**